Tema6/src/main/java/SetString.java

```diff
@@ -0,0 +1,54 @@
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+
+public class SetString {
+
+    Set<String> setRandom = new HashSet<>();
+
+    public String randomString() {
```

Comment:
generateRandomString (please verify again the naming convetions, they are pretty important)